### PR TITLE
fix: only allow btc usd as traiding pair

### PR DESCRIPTION
**Pull request by holzeis**
We currently don't support ETHUSD, hence removing the option for now,

### CODE REVIEW

File: lib/cfd_trading/cfd_offer.dart [hunk 1]
```diff
@@ -95,7 +95,9 @@ class _CfdOfferState extends State<CfdOffer> {
             value: order.quantity.toString(),
           ),
           Dropdown(
-              values: TradingPair.values.map((t) => t.name.toUpperCase()).toList(),
+              // Only BTCUSD is currently supported, hence I am removing the list here for now.
```
Comment (da-kami) [inline lib/cfd_trading/cfd_offer.dart:98]:
```suggestion
```

File: lib/cfd_trading/cfd_offer.dart [hunk 2]
```diff
@@ -95,7 +95,9 @@ class _CfdOfferState extends State<CfdOffer> {
...
+              // TradingPair.values.map((t) => t.name.toUpperCase()).toList(),
```
Comment (da-kami) [inline lib/cfd_trading/cfd_offer.dart:99]:
```suggestion
```

**Review by da-kami — APPROVED** [review]
Happy to merge this. I would remove the comments.

Note: We could instead add "coming soon" messages when we implement the validation in #200 - but probably easier to just remove it, similar to restricting the leverage

**Review by klochowicz — APPROVED** [review]

**Pull request merged**